runtime/src/lib.rs

```diff
@@ -253,6 +253,23 @@ impl vln_transfers::Config for Runtime {
     type Assets = Tokens;
 }
 
+parameter_types! {
+    pub const MinimumCount: u32 = 3;
+    pub const ExpiresIn: u32 = 600;
+    pub RootOperatorAccountId: AccountId = Sudo::key();
+}
+
+impl orml_oracle::Config for Runtime {
+    type Event = Event;
+    type OnNewData = ();
+    type CombineData = orml_oracle::DefaultCombineData<Runtime, MinimumCount, ExpiresIn>;
+    type Time = Timestamp;
+    type OracleKey = Asset;
+    type OracleValue = u64;
```

Comment:
Wonder if this will work in the future or if we'll need a fixed point type 🤔

Reply:
I've changed it to [FixedU128](https://docs.rs/sp-runtime/2.0.0-rc4/sp_runtime/struct.FixedU128.html) to align with [Acala](https://github.com/AcalaNetwork/Acala/blob/2a445fe0b69ede82833acdba8c5213cf55579f82/modules/support/src/lib.rs#L25)